pkg/controller/client.go

```diff
@@ -16,7 +16,7 @@ func (c *Controller) PatchStatus(ctx context.Context, patch *v1alpha1.SpiceDBClu
 	}
 	patch.ManagedFields = nil
 	patch.Status.ObservedGeneration = patch.Generation
-	data, err := client.Apply.Data(patch)
+	data, err := json.Marshal(patch)
```

Comment:
any semantic differences between this and the only `Apply` stuff?

Reply:
nope: https://github.com/kubernetes-sigs/controller-runtime/blob/304027bcbe4b3f6d582180aec5759eb4db3f17fd/pkg/client/patch.go#L212